presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorOutputMetadata.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.connector;
+
+public interface ConnectorOutputMetadata
```

Comment:
This is weird to have an interface with no methods that isn't used any type-specific way (unlike, say, handles). The only requirement is that the returned object be Jackson-serializable. So why not just return `Object`? That is more consistent with `ConnectorSplit.getInfo()`